staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go

```diff
@@ -158,8 +158,17 @@ func ruleMatchesResource(r *audit.PolicyRule, attrs authorizer.Attributes) bool
 				return true
 			}
 			for _, res := range gr.Resources {
-				if res == resource {
-					if len(gr.ResourceNames) == 0 || hasString(gr.ResourceNames, name) {
+				if len(gr.ResourceNames) == 0 || hasString(gr.ResourceNames, name) {
+					// match '*'
+					if res == combinedResource || res == "*" {
+						return true
+					}
+					// match '*/subresource'
```

Comment:
only if subresource is non-empty, right?

Reply:
Yes, need to make sure subresource non-empty. I will do it now.

Reply:
@liggitt Thanks for pointing out this. Fixed now.